CHANGES_NEXT_RELEASE

```diff
@@ -16,3 +16,4 @@
 - Add: subscriptions triggered by modifications in any attribute without explicitly list them (aka ONANYCHANGE) (Issue #350)
 - Add: 'q' expression evaluation in NGSIv2 subscription (at subscription creation/update and update context times) (Issue #1316 and #1658)
 - Fix: subscription service path is lost at update subscription time (Issue #1693)
+- Add: Log summary (Issue #1585)
```

Comment:
ok.
I was never really satisfied with that line. Also not in cli.md.
The problem is that you both turn on log summary and set the interval with one single option.